Sources/PodToBUILD/SwiftLibrary.swift

```diff
@@ -255,6 +255,7 @@ public struct SwiftLibrary: BazelTarget {
                 .named(name: "copts", value: coptsSkylark),
                 .named(name: "swiftc_inputs", value: swiftcInputs.toSkylark()),
                 .named(name: "generated_header_name", value: (externalName + "-Swift.h").toSkylark()),
+                .named(name: "generates_header", value: SkylarkNode.int(1)),
```

Comment:
I wonder if there should be a `SkylarkNode.bool` that generates `True` / `False` for nicer readability.

Reply:
I was looking for that, will try!